test/l2-strategies.js

```diff
@@ -286,9 +286,165 @@ const createAaveFLV3BoostL2Strategy = () => {
     return aaveV3BoostL2Strategy.encodeForDsProxyCall();
 };
 
+const aaveV3CloseActions = {
+    flAction: () => new dfs.actions.flashloan.AaveV3FlashLoanAction(
+        ['%repayAmount'], // cant pipe in FL actions :(
+        ['%debtAsset'],
+        ['%AAVE_NO_DEBT_MODE'],
+        '%nullAddress',
+    ),
+
+    paybackAction: () => new dfs.actions.aaveV3.AaveV3PaybackAction(
+        '%true', // useDefaultMarket - true or will revert
+        '&nullAddress', // market
+        '%repayAmount', // kept variable (can support partial close later)
+        '&proxy',
+        '%rateMode',
+        '&debtAsset', // one subscription - one token pair
+        '&debtAssetId',
+        '%false', // useOnBehalf - false or will revert
+        '&nullAddress', // onBehalfOf
+    ),
+
+    withdrawAction: () => new dfs.actions.aaveV3.AaveV3WithdrawAction(
+        '&collAssetId', // one subscription - one token pair
+        '%true', // useDefaultMarket - true or will revert
```

Comment:
If we hardcode this no need to hardcode &nullAddress later, thats how we did it in other aaveV3 strategies